pkg/authorization/api/v1/conversion.go

```diff
@@ -12,7 +13,7 @@ import (
 	uservalidation "github.com/openshift/origin/pkg/user/api/validation"
 )
 
-func convert_v1_ResourceAccessReview_To_api_ResourceAccessReview(in *ResourceAccessReview, out *newer.ResourceAccessReview, s conversion.Scope) error {
+func Convert_v1_ResourceAccessReview_To_api_ResourceAccessReview(in *ResourceAccessReview, out *newer.ResourceAccessReview, s conversion.Scope) error {
```

Comment:
The generators look up the function by its function pointer, then ask Go
what the name of the function is (and what package is it in).  If it has a
public name it is used.  So hand written conversions can have arbitrary
names.

On Tue, Mar 8, 2016 at 6:49 AM, David Eads <notifications@github.com> wrote:

> In pkg/authorization/api/v1/conversion.go
> <https://github.com/openshift/origin/pull/7814#discussion_r55354315>:
>
> > @@ -12,7 +13,7 @@ import (
> >  	uservalidation "github.com/openshift/origin/pkg/user/api/validation"
> >  )
> >
> > -func convert_v1_ResourceAccessReview_To_api_ResourceAccessReview(in *ResourceAccessReview, out *newer.ResourceAccessReview, s conversion.Scope) error {
> > +func Convert_v1_ResourceAccessReview_To_api_ResourceAccessReview(in *ResourceAccessReview, out *newer.ResourceAccessReview, s conversion.Scope) error {
>
> Do these names still have to match the format exactly or did we update the
> generation code to a reflected check?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7814/files#r55354315>.
>